src/demo-app/list/list-demo.html

```diff
@@ -22,6 +22,7 @@ <h3 mat-line>{{contact.name}}</h3>
       </mat-list-item>
     </mat-list>
 
+<<<<<<< HEAD
```

Comment:
Did you forgot to remove this? Also in line 43.

Reply:
Resolved, thanks for the catch. I rebased and for some reason Git missed this.